examples/tensorflow-akka/src/main/scala/modelserving/wine/WineModelServer.scala

```diff
@@ -0,0 +1,51 @@
+package modelserving.wine
+
+import cloudflow.akkastream.AkkaStreamlet
+import cloudflow.akkastream.scaladsl.{ FlowWithOffsetContext, RunnableGraphStreamletLogic }
+import cloudflow.streamlets.avro.{ AvroInlet, AvroOutlet }
+import cloudflow.streamlets.{ ReadWriteMany, StreamletShape, StringConfigParameter, VolumeMount }
+
+import modelserving.wine.avro._
+
+/**
+ * Scores WineRecords using a TensorFlow model that is read from a volume mount.
+ * The model used can be changed by deploying or reconfiguring this streamlet using `kubectl cloudflow deploy`
+ * or `kubectl cloudflow configure`.
+ */
+final class WineModelServer extends AkkaStreamlet {
+
+  val in = AvroInlet[WineRecord]("wine-records")
+  val out = AvroOutlet[WineResult]("wine-results", _.inputRecord.datatype)
```

Comment:
should we encourage plurals (`wine-records`, `wine-results`) as the port names ?

Reply:
yeah that would be my personal preference, when you try to make it more clear what the 'entities' are that will be read and written.